scripts/zones/Promyvion-Vahzl/mobs/Provoker.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Promyvion - Vahzl
+--   NM: Provoker
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/magic")
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:addMod(tpz.mod.ACC, 50)
+    mob:setMobMod(tpz.mobMod.IDLE_DESPAWN, 180)
+    mob:setMobMod(tpz.mobMod.ADD_EFFECT, 1)
+end
+
+function onMobFight(mob, target)
+
+    local changeTime = mob:getLocalVar("changeTime")
+    local element = mob:getLocalVar("element")
+
+    if (changeTime == 0) then
```

Comment:
If there's only simple logic going on, please drop the brackets
```lua
if changeTime == 0 then
    ...
```

Same for the rest of the file etc.

Reply:
You're almost there, there are still some simple logic blocks (like that while loop) that don't need brackets. Then we're good to go

Reply:
Sorry but are there actually any more unnecessary brackets aside from the while loop? I'm not too familiar with which other brackets may be optional in Lua quite yet, and I don't want to make a bunch of separate commits because I missed something else!